### ISSUE TITLE
Problems is a maintenance problem

### ISSUE THREAD

**Issue by lindsayad** [arfc/moltres#88]
The `problems` directory is a maintenance problem. Users come and use Moltres and they use input files in that directory as jumping off points for their work, which is great. The problem is that at least some of them (probably most) won't run with modern Moltres/MOOSE. We should at least be running syntax checks on any input file in that directory (and sub-directory) in order to ensure that a new user who comes along and wants to run one of the inputs will actually be able to run the input.

Any untested input file is a ticking bomb for failure. I'm the one to blame...but I also doubt I will have time to address this.

In summary, every git tracked file should be getting touched by a test, even if it is just a syntax test.

**Comment by katyhuff** [comment]
I agree. I was thinking the same (see https://github.com/arfc/moltres/issues/87)

**Comment by katyhuff** [comment]
(and -- I'll have a handful of undergrads this summer, and we have a handful of grads starting to have moltres responsibilities... @smpark7 @achubbz @robfairh @joonhon @dogrady2 ... so hopefully we can get some folks to dedicate a little time to this. shouldn't take long, but might be a couple weeks from now before it's done.)

**Comment by lindsayad** [comment]
The github issue suggester should have done a better job :-) closing in favor of #87
